src/modules/FileLocksmith/FileLocksmithLibInterop/NtdllExtensions.cpp

```diff
@@ -309,9 +309,8 @@ std::wstring NtdllExtensions::pid_to_user(DWORD pid)
     }
 
     DWORD token_size = 0;
-    GetTokenInformation(token, TokenUser, nullptr, 0, &token_size);
-
-    if (token_size < 0)
+    const bool ok = GetTokenInformation(token, TokenUser, nullptr, 0, &token_size);
+    if (!ok && (GetLastError() != ERROR_INSUFFICIENT_BUFFER || !token_size))
```

Comment:
```suggestion
    if ((!ok && GetLastError() != ERROR_INSUFFICIENT_BUFFER) || !token_size)
```
Would this make more sense? We sort of don't want to allocate a 0-sized buffer even if GetTokenInformation is OK.

Reply:
It's currently done according to this documentation paragraph, but it should be fine either way.

> A pointer to a variable that receives the number of bytes needed for the buffer pointed to by the TokenInformation parameter. If this value is larger than the value specified in the TokenInformationLength parameter, the function fails and stores no data in the buffer.